data/tsv/lat_latn_vulg_phonemic.tsv

```diff
@@ -1,536 +1,3 @@
-*abanteō	a β a n t e oː
-*abbatuō	a β β a t u oː
-*abbatō	a β β a t oː
-*abbiberō	a β β i β e r oː
-*absedium	a p s ɛ d i u m
-*absediī	a p s ɛ d i iː
-*abyssimus	a β e s s i m u s
```

Comment:
Prefix asterisks in Latin entries (here and in the Ecclesiastical version) refer to reconstructed words. Again, not clear what's the way going forward with those.

Reply:
I wouldn't say that's WAI exactly but it's a nice side-effect. Please keep it that way!